static/js/timerender.js

```diff
@@ -146,7 +146,7 @@ function render_date_span(elem, rendered_time, rendered_time_above) {
         return elem;
     }
     elem.append(_.escape(rendered_time.time_str));
-    return elem.attr("title", rendered_time.formal_time_str);
+    return elem.attr("data-tippy-content", rendered_time.formal_time_str);
```

Comment:
This, corresponding topic title bar(June 10 in image) isn't showing a tooltip but just showing a text select cursor(`I`). Not sure how to fix this. Will try. Changing it back to `title` for now.
![image](https://user-images.githubusercontent.com/44665669/122448748-b9a2ee80-cfc2-11eb-9cc6-aaa1ed289ce9.png)
@amanagr Do you have any ideas here? This is the code I placed here:
```
    elem.addClass("tippy-zulip-tooltip");
    elem.attr("data-tippy-placement", "top");
    return elem.attr("data-tippy-content", rendered_time.formal_time_str);
```

Reply:
https://github.com/zulip/zulip/pull/18856#discussion_r653068735 will fix it.

Reply:
The `elem` is being created through js there. I also tried adding it to the parent in the template and setting those `attr`s to parent, but it didn't work.

I'm not sure how we add the class changes the behaviour. It worked perfectly for the message timestamp this way.

FWIW, the `div` that tippy adds is being added to the page but it's just not visible as shown in below screenshot:
![image](https://user-images.githubusercontent.com/44665669/122594785-1e268200-d085-11eb-8e7f-ab252576c3c6.png)

I wonder if this has something to do with CSS.

(I think we can merge this PR for now as this PR is mainly focused on message timestamp.)

Reply:
One theory is that the `timerender` elements get live-updated when e.g. the date changes, and maybe that live-update logic isn't working properly.  We should investigate that possibility; when you change `data-tippy-content`, that doesn't update the popovers logic properly, and you need to do an `instance.setContent` (I think).

Reply:
It actually seems to be some CSS thing of `.recipient_row_date` or its parents but I'm not sure what property of those is keeping this tooltip hidden.

Using `appendTo: () => document.body,` in tippy configurations renders it. This also renders the tooltip in centre when right sidebar is present for the timestamp. So, I think we can use `appendTo: () => document.body` for both.

Reply:
Sounds good!